app/feature/feature-profile/src/main/kotlin/com/hedvig/android/feature/profile/tab/ProfileDestination.kt

```diff
@@ -183,29 +181,27 @@ private fun ProfileScreen(
       if (uiState is ProfileUiState.Success) {
         val memberReminders =
           uiState.memberReminders.onlyApplicableReminders(notificationPermissionState.status.isGranted)
+        val padding = PaddingValues(horizontal = 16.dp) + WindowInsets.safeDrawing
+          .exclude(consumedWindowInsets)
+          .only(WindowInsetsSides.Horizontal)
+          .asPaddingValues() // todo: check if the padding is right
         MemberReminderCards(
           memberReminders = memberReminders,
           navigateToConnectPayment = navigateToConnectPayment,
           navigateToAddMissingInfo = navigateToAddMissingInfo,
           openUrl = openUrl,
           notificationPermissionState = notificationPermissionState,
           snoozeNotificationPermissionReminder = snoozeNotificationPermission,
-          contentPadding = PaddingValues(horizontal = 16.dp) + WindowInsets.safeDrawing
-            .exclude(consumedWindowInsets)
-            .only(WindowInsetsSides.Horizontal)
-            .asPaddingValues(),
+          contentPadding = padding,
           modifier = Modifier.onConsumedWindowInsetsChanged { consumedWindowInsets = it },
           onNavigateToNewConversation = onNavigateToNewConversation,
         )
         if (memberReminders.isNotEmpty()) {
           Spacer(Modifier.height(16.dp))
         }
       }
-      HedvigTextButton(
+      HedvigRedTextButton(
```

Comment:
This used to also go full width, while now it's a small center-aligned button.
Do we perhaps want to use the big version of this button here, or is the new design specifically asking for a small button here?

Reply:
No, I don't think so. Changing it to Large.